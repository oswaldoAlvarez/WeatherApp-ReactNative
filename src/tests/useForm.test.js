import React from 'react';
import renderer from 'react-test-renderer';

test('Probando funciones del useForm', () => {
  const tree = renderer.create(<useForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
