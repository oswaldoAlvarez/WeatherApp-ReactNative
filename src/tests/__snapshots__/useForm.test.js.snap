// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando funciones del useForm 1`] = `<useForm />`;
